cvat-core/src/server-response-types.ts

```diff
@@ -495,3 +495,26 @@ export interface SerializedAPISchema {
         url: string;
     };
 }
+
+export interface SerializedRequest {
+    id?: string;
```

Comment:
why having `id` is not guaranteed? 

tha same about `created_date` and `owner`

Reply:
Yes. When we create task, we use the same request callback, so we create fake Request instances. We dont have an ID while uploading data, also it wouldnt be pretty to construct a created_date and owner here.